server/cluster/ad/adPlugin.ts

```diff
@@ -105,6 +105,21 @@ export default function adPlugin(Client: any, config: any, components: any) {
     },
     method: 'GET',
   });
+
+  ad.searchResults = ca({
```

Comment:
is it ok that this has the same name as the one implemented directly below? Usual strategy is to name to something more verbose - for example, the start detector has 2 different calls, one for real-time (`ad.startDetector`) and one for historical (`ad.startHistoricalDetector`)

Reply:
Actually we don't need this one, as the result index is not required, check line 115 `required: false,`. 
Removed the second `ad.searchResults` and tested, everything looks good.